docs/LOCAL_DEVELOPMENT.md

```diff
@@ -1,24 +1,158 @@
 # Local Development
 
-> **This guide is not yet complete.**
+This guide provides instructions for setting up and working with the Time-addressable Media Store project in a local development environment.
 
 ## Pre-Requisites
 
-The following dependencies must be installed. Please refer to your operating system, how to install them.
+The following dependencies must be installed. Please refer to your operating system documentation for installation instructions.
 
-> **Note:** For Windows, we recommend enabling Windows Subsystem for Linux (WSL) and installing Linux distribution of your choice,
+> **Note:** For Windows, we recommend enabling Windows Subsystem for Linux (WSL) and installing a Linux distribution of your choice,
 > for example, here are the instructions on how to install [Ubuntu](https://ubuntu.com/tutorials/ubuntu-on-windows).
 
 - Python ~= 3.13.0 and pip
-- [poetry](https://python-poetry.org/)
-- [pre-commit](https://pre-commit.com/#install)
-- [cfn-lint](https://github.com/aws-cloudformation/cfn-lint)
-- [cfn-nag](https://github.com/stelligent/cfn_nag)
+- [poetry](https://python-poetry.org/) - Python dependency management
+- [pre-commit](https://pre-commit.com/#install) - Git hooks manager
+- [cfn-lint](https://github.com/aws-cloudformation/cfn-lint) - CloudFormation linting
+- [cfn-nag](https://github.com/stelligent/cfn_nag) - CloudFormation security scanning
+- [AWS SAM CLI](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install.html) - Serverless Application Model CLI
+- [Docker](https://hub.docker.com/search/?type=edition&offering=community) - Required for local Lambda function testing
+- [rain](https://github.com/aws-cloudformation/rain) - CloudFormation template formatting
+- [git-secrets](https://github.com/awslabs/git-secrets) - Prevents committing secrets
```

Comment:
I think pre-commit will take care of installing this, so I'm not sure it needs to be mentioned here.

Reply:
I meant for git-secrets. (rain too, but I noticed there's a Makefile target that uses rain and that won't use the pre-commit version). Not sure why github marked the other lines.